plugins/core/Standard/qMeshBoolean/src/qMeshBoolean.cpp

```diff
@@ -39,12 +39,12 @@
 #endif
 
 //libIGL
+#include <igl/copyleft/cgal/mesh_boolean.h>
```

Comment:
To respect the spirit of the warnings disabling lines below, you should move the #include AFTER the #ifdef _MSC_VER section (otherwise the #pragma warning calls will be useless).